parameters/world_modeling/world_model_learn_objects.yaml

```diff
@@ -0,0 +1,28 @@
+world_name: learn_objects
```

Comment:
This file is part of a project to automate object learning. After the evaluation we decided that it would be better to purchase a second camera to use for object learning, making this project obsolete.
I would suggest to remove this project entirely, or to move it to a separate branch.

Reply:
As mentioned above. Keep it until we actually switched.